scripts/userstyles.yml

```diff
@@ -469,6 +484,7 @@ userstyles:
   migadu-webmail:
     name: Migadu Webmail
     categories: [email_client, productivity]
+    color: text
```

Comment:
```suggestion
    color: blue
```
https://www.migadu.com/svg/logo.svg:
![](https://www.migadu.com/svg/logo.svg)

Reply:
I'm trying to keep it in line with simple icons

Reply:
It will be blue soon: https://github.com/simple-icons/simple-icons/issues/10498

Reply:
We can raise a PR after its updated :+1: